lib/containers/utils.pm

```diff
@@ -179,4 +179,22 @@ sub test_built_img {
     assert_script_run("rm -rf /root/templates");
 }
 
+=head2 can_build_sle_base
+
+C<can_build_sle_base> should be used to identify if sle base image runs against a
+system that it does not support registration and SUSEConnect B<for job running on OSD>.
+In this case the build of the container engine is not going to work.
+
+The call should return false if you try to test sle base container on osd but host is not sle
+true otherwise.
+
+=cut
+sub can_build_sle_base {
+    my $is_osd = get_var('OPENQA_HOSTNAME', '') =~ /suse.de|suse.cz/;
+    return 0 if $is_osd && check_os_release('opensuse', 'PRETTY_NAME');
+    return 0 if $is_osd && check_os_release('ubuntu',   'PRETTY_NAME');
+    return 0 if $is_osd && check_os_release('centos',   'PRETTY_NAME');
+    return 1;
```

Comment:
```suggestion
    return 1 if is_sle();
```
This should be enough.

Reply:
This was requested change already approved. `is_sle` was not the best option. the reason is because it cannot distinguish whether opensuse runs in osd